api/api.proto

```diff
@@ -371,6 +371,11 @@ message VolumeSpecUpdate {
   // `ownership` message is an empty string then the value of `owner` in
   // the `VolumeSpec.Ownership.owner` will not be updated.
   Ownership ownership = 26;
+  // Nodiscard specifies if the volume will be mounted with discard support disabled.
```

Comment:
These values look good. You will need to update the SdkVersion and the SDK_CHANGELOG.md

Reply:
Updated.